### PR TITLE
Delivery password modales

### PR DESCRIPTION
**Pull request by Hugo-Hontans**

### CODE REVIEW

**Comment by pldespaigne** [comment]
Everything seems ok, but since it is a big PR a second opinon is needed

File: libs/material/src/lib/delivery/delivery-sign/delivery-sign.component.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<div mat-dialog-content>
+  <h2 fxLayoutAlign="center">
+    Please confirm with your password.
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/delivery-sign/delivery-sign.component.html]: "Please enter your password to confirm" seems to be a more viable syntax.

File: libs/material/src/lib/delivery/delivery-sign/delivery-sign.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      {
+        onConfirm: () => void
+      },
+    public dialogRef: MatDialogRef<DeliverySignComponent>,
+    private snackBar: MatSnackBar,
+    ) {}
+
+  public sign() {
+    this.data.onConfirm();
+    this.snackBar.open('Validated delivery', 'close', { duration: 2000 });
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/delivery-sign/delivery-sign.component.ts]: "Delivery approved" or "Delivery signed" to match with our vocabulary pattern.

File: libs/material/src/lib/delivery/delivery.module.ts [hunk 3]
```diff
@@ -50,11 +56,7 @@ import {
 } from '@angular/material';
 import { MatFormFieldModule } from '@angular/material';
 import { ConfirmModule, UiFormModule } from '@blockframes/ui';
-import { TeamWorkActionsComponent } from './delivery-team-work-view/team-work-actions/team-work-actions.component';
-import { TeamWorkFiltersComponent } from './delivery-team-work-view/team-work-filters/team-work-filters.component';
-import { DeliverySettingsItemComponent } from './delivery-settings-item/delivery-settings-item.component';
-import { DeliverySettingsFormComponent } from './delivery-settings-form/delivery-settings-form.component';
-import { DeliverySettingsViewComponent } from './delivery-settings-view/delivery-settings-view.component';
+import { DeliveryEmptyComponent } from './delivery-empty/delivery-empty.component';
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/delivery.module.ts]: This import belongs to ```// Components``` section

File: libs/material/src/lib/delivery/stakeholder-list/stakeholder-list.component.ts [hunk 4]
```diff
@@ -29,12 +32,22 @@ export class StakeholderListComponent implements OnInit, OnDestroy {
...
+      }
+    });
+  }
+
   public signDelivery() {
     this.service.signDelivery();
   }
 
+
   public navigateToSettings() {
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/stakeholder-list/stakeholder-list.component.ts]: Rename it to ```navigateToTeamwork()``` to match the landing path.

File: libs/movie/src/lib/movie/home/home.component.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-
+<movie-home-empty *ngIf="!(hasOrgs$ | async)"></movie-home-empty>
```
Comment (ioaNikas) [inline libs/movie/src/lib/movie/home/home.component.html]: Indentation seems strange

File: libs/movie/src/lib/movie/home/home.component.ts [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
 import { AuthQuery } from '@blockframes/auth';
 import { Observable } from 'rxjs';
+import { OrganizationQuery } from '@blockframes/organization';
 
 @Component({
 // tslint:disable-next-line: component-selector
```
Comment (ioaNikas) [inline libs/movie/src/lib/movie/home/home.component.ts]: Why disable ?  "movie" is a part of selectors in tslint.json

**Pull request merged**